src/components/TransactionsTable.tsx

```diff
@@ -60,20 +60,24 @@ function TrasactionsTable({
     return (
       <span style={{ whiteSpace: 'nowrap' }}>
         <Button
-          style={{ marginRight: 10 }}
-          type="ghost"
+          style={{ marginRight: 10, padding: '3px 16px 4px 16px' }}
+          type="primary"
           shape="round"
           onClick={() => selectRoute(route)}>
           Resume Swap
         </Button>
         <Popconfirm
           title={<>Are you sure to delete this transfer?</>}
           onConfirm={() => deleteRoute(route)}
-          // onCancel={cancel}
```

Comment:
Can this be removed?

Reply:
what about the `cancel` function? Can it be removed as well?

Reply:
yep